app/Models/Pharmacy.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Pharmacy extends Model
+{
+    use HasFactory;
+
+    protected $guarded = [];
+
+    public function path()
+    {
+        return "/pharmacies/{$this->id}";
```

Comment:
This is a bit risky, as you might decide to use a different path for the pharmacies, or you might introduce localization and have a 'farmakeia/id' version. Use the route helper instead, to create a full path URL that will adapt.
```suggestion
        return route('pharmacies.show', $this->id);
```